src/components/Expenses/ExpenseDate.css

```diff
@@ -0,0 +1,26 @@
+.expense-date {
+  display: flex;
+  flex-direction: column;
+  width: 5.5rem;
+  height: 5.5rem;
```

Comment:
width and height is the same. wouldn't it be nice to have a variable here?
since you don't use any css pre processors maybe you can use native css variables?
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties